src/main/java/dev/codesoapbox/dummy4j/dummies/shared/valueobject/Address.java

```diff
@@ -0,0 +1,88 @@
+package dev.codesoapbox.dummy4j.dummies.shared.valueobject;
+
+import dev.codesoapbox.dummy4j.dummies.shared.string.StringValidator;
+
+import java.util.*;
+
+/**
+ * A value object representing an address
```

Comment:
`representing a complete address`? Since we're throwing errors if any of the properties are empty.